graphql-proxy/.env.example

```diff
@@ -1,5 +1,5 @@
-DEBUG=debug
 NODE_ENV=development
-PORT=4000
-ENGINE_API_KEY=tapahtumat
-API_BASE_URL=http://localhost:4000
+GRAPHQL_PROXY_DEBUG=debug
+GRAPHQL_PROXY_PORT=4000
+GRAPHQL_PROXY_ENGINE_API_KEY=tapahtumat
+GRAPHQL_PROXY_API_BASE_URL=http://localhost:4000
```

Comment:
Is this file actually used? The same variables are defined in the top-level `.env.example` (except for `NODE_ENV`).